scripts/baseline_view.py

```diff
@@ -152,11 +153,11 @@ def update_table(self):
     self._table_list = self.table.items()
 
   def gps_time_callback(self, data):
-    self.week = sbp_messages.GPSTime(data).wn
-    self.nsec = sbp_messages.GPSTime(data).ns
+    self.week = MsgGPSTime(data).wn
```

Comment:
The contract on per-message-type callbacks has been updated to take the entire message - updates have been made where appropriate to use `.payload` or to use the entire message, as here.